tests/unit/utils/test_fs.py

```diff
@@ -149,3 +150,10 @@ def test_relpath_windows_different_drives():
     rel_info = relpath(info1, info2)
     assert isinstance(rel_info, str)
     assert rel_info == path1
+
+
+def test_get_inode(repo_dir, dvc_repo):
```

Comment:
You don't need `dvc_repo` fixture, since you are not using it anyways.
```suggestion
def test_get_inode(repo_dir):
```